src/lib/coapcore/Cargo.toml

```diff
@@ -44,6 +51,8 @@ document-features = "0.2.10"
 ccm = { version = "0.5.0", default-features = false }
 aes = { version = "0.8.4", default-features = false }
 
+p256 = { version = "0.13.2", features = ["ecdsa"], default-features = false }
```

Comment:
This should only get pulled in with code when there is a token assigned; I should verify that.

Reply:
The compiler currently does not see that -- it might (because there is only one thing that writes anything other than None into `ConfigBuilder.as_key_neg7`, and if that doesn't happen, that branch is never taken), but it seems it doesn't do that right now.

I have a pending item to make more crypto stuff optional; if it's OK with you, I'll do that there rather than starting to half-do it here.